src/components/ReportActionItem/TaskPreview.js

```diff
@@ -59,7 +65,8 @@ function TaskPreview(props) {
         ? props.taskReport.stateNum === CONST.REPORT.STATE_NUM.SUBMITTED && props.taskReport.statusNum === CONST.REPORT.STATUS.APPROVED
         : props.action.childStateNum === CONST.REPORT.STATE_NUM.SUBMITTED && props.action.childStatusNum === CONST.REPORT.STATUS.APPROVED;
     const taskTitle = props.taskReport.reportName || props.action.childReportName;
-    const taskAssignee = props.taskReport.managerEmail || props.action.childManagerEmail;
+    const taskAssigneeAccountID = TaskUtils.getTaskAssigneeAccountID(props.taskReport);
```

Comment:
I don't think so - but since we were defaulting to `managerEmail` first, I thought it would be best if we default to `managerID` first as well, THEN use `childManagerAccountID` - and that logic exists in this function so I thought it would be better to reuse it, do you disagree?

Reply:
"original logic" being `props.taskReport.managerEmail || props.action.childManagerEmail;`? We're trying to get rid of those :D